mapdata/WarcraftLegacies/AbilityData/Skin/1245271123.json

```diff
@@ -0,0 +1,119 @@
+{
+  "OldId": 1634693697,
+  "NewId": 1245271123,
+  "Unk": [
+    0
+  ],
+  "Modifications": [
+    {
+      "Id": 2020631137,
+      "Value": 3
+    },
+    {
+      "Id": 1718840929,
+      "Type": 3,
+      "Value": "(tess)"
+    },
+    {
+      "Id": 1835101793,
+      "Type": 3,
+      "Value": "Vanish"
+    },
+    {
+      "Id": 1918988897,
+      "Type": 3,
+      "Value": "ReplaceableTextures/CommandButtons/BTNAbility_Vanish.blp"
+    },
+    {
+      "Id": 2020635233,
+      "Value": 3
+    },
+    {
+      "Id": 1802007137,
+      "Type": 3,
+      "Value": "R"
+    },
+    {
+      "Id": 1952805473,
+      "Type": 3,
+      "Value": "Learn Vanish- [|cffffcc00Level %d|r]"
+    },
+    {
+      "Id": 1953854049,
+      "Type": 3,
+      "Value": "The hero vanishes, becoming invisible and gains 20 bonus damage. The invisibility will not dissipate even if the hero attacks or casts spells.|n|n|cffffcc00Level 1|r - 10 seconds duration|n|cffffcc00Level 2|r - 15 seconds duration|n|cffffcc00Level 3|r - 20 seconds duration"
```

Comment:
All of the numbers here should be generated from the data.

Reply:
can you generate numbers from code???

Reply:
No - isn't the 20 bonus damage just an ability that gets temporarily applied to Tess?
But yes you're right, the duration can't be generated.